Tests/test_coordinates.py

```diff
@@ -0,0 +1,16 @@
+import unittest
+from Models.coordinates import Coordinates
+
+
+class CoordinatesUnitTests(unittest.TestCase):
+    def test_compute_distance_to_returns_expected_value(self):
```

Comment:
:+1: for testing the things that matter, I hate it when I see tests asserting setters and getters